packages/website/src/components/Uploader/Uploader.js

```diff
@@ -201,7 +201,7 @@ const Uploader = () => {
             response = await client.uploadFile(file, { onUploadProgress });
           }
 
-          const url = client.getSkylinkUrl(response.skylink, { subdomain: mode === "directory" });
+          const url = await client.getSkylinkUrl(response.skylink, { subdomain: mode === "directory" });
```

Comment:
I don't think you caught all occurrences. This is also called from the `Dashboard` and from an index.js file somewhere.

Reply:
I only updated website package, dashboard is a separate one